### PR TITLE
fix: Hints content revision for next release

### LINKED ISSUE #326: Hints content revision for next release
Please go through the pages:

- [x] dashboard
- [x] state channel details
- [x] state channel
- [x] transactions
- [x] token details
- [x] tokens
- [x] oracle details
- [x] oracles
- [x] account details
- [x] name details
- [x] names
- [x] smart contracts

and evaluate the content.

Ideally, provide comments to this Issue.

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #326 

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] fix content by review

### CODE REVIEW

File: src/utils/adapters.js [hunk 1]
```diff
@@ -54,6 +54,7 @@ export function adaptTransactions(transactions) {
 }
 
 export function adaptContracts(contracts) {
+  console.log('contracts', contracts)
```
Comment (michele-franchi) [inline src/utils/adapters.js:57]: There are some `console.logs`, could you please check it and remove them?

**Review by michele-franchi — CHANGES_REQUESTED** [review]
Thanks for adjusting them, I found some issues:

- In the dashboard the "Participants" hint is missing:
image

- OracleQueryTx in the transactions table should be: "An oracle can be queried by any account and requires the oracle fee to be provided. In case the oracle operator does not respond in time, the oracle fee will be returned." 

- NameUpdateTx in the transactions table should be: "Can be used to extend the lifetime of a name and/or to update the pointers of a name. Extending the lifetime of a name is free and does not require any extra fee."

- ChannelCloseSoloTx in the transactions table should be: "A solo close has to be settled via a ChannelSettleTx after the lock period has passed. The lock period is being agreed upon when opening the channel."

- There's no price in the name detail so I would keep the old hint "Owner of the name" until we add it.

image

- For active names, in name details "Ends" should be "Expires" and the hint should be: "Keyblock height and estimated date and time when the name will be detached from the account. After expiration, the name enters into a revoked state and can be claimed again after a fixed timeout of 2016 keyblocks."

- Buyer hint in the actives names table should be: "Account that paid the shown price to claim the name."
image

- Expires hints in the active names table should be: "Keyblock height and estimated date and time when the name will be detached from the account. After expiration, the name enters into a revoked state and can be claimed again after a fixed timeout of 2016 keyblocks."
image

- On mobile "Bid count" in the "In Auctions" names table is incorrect:
image

**Comment by janmichek** [comment]
> In the dashboard the "Participants" hint is missing:

It was suggested to be removed

https://github.com/aeternity/aescan/issues/326#issuecomment-1596903613

File: src/utils/adapters.js (hunk 1 above)
Comment (janmichek) [reply src/utils/adapters.js]: ooops, removed

**Comment by michele-franchi** [comment]
> > In the dashboard the "Participants" hint is missing:
> 
> It was suggested to be removed
> 
> [#326 (comment)](https://github.com/aeternity/aescan/issues/326#issuecomment-1596903613)

The suggested hint to be remove is the State Channel one, not the participants.

File: src/components/AccountTransactionsTable.vue [hunk 2]
```diff
@@ -60,6 +68,7 @@ import DatetimeLabel from '@/components/DatetimeLabel'
 import ValueHashEllipsed from '@/components/ValueHashEllipsed'
 import { accountHints } from '@/utils/hints/accountHints'
 import HintTooltip from '@/components/HintTooltip'
+import { transactionsHints } from '~/utils/hints/transactionsHints'
```
Comment (michele-franchi) [inline src/components/AccountTransactionsTable.vue:71]: I would change it to:

```suggestion
import { transactionsHints } from '@/utils/hints/transactionsHints'
```

**Review by michele-franchi — APPROVED** [review]
Good job, looks perfect now! 🚀 

Please have a look at the import change I suggested.

File: src/components/AccountTransactionsTable.vue (hunk 2 above)
Comment (janmichek) [reply src/components/AccountTransactionsTable.vue]: right, fixed

Comment (janmichek) [reply src/components/AccountTransactionsTable.vue]: It was present on more places, also fixed

**Pull request merged**